### PR TITLE
Add RPCFast endpoints

### PR DESCRIPTION
**Pull request by VladStarr**
If you are adding a new RPC, please answer the following questions.

#### Link the service provider's website (the company/protocol/individual providing the RPC):

https://rpcfast.com

#### Provide a link to your privacy policy:

https://rpcfast.com/privacy

### CODE REVIEW

**Review by wlanroot — APPROVED** [review]
4f2T2SmrWhGLkGge2qpMQi5qP4Ldg5rkwpkg4sQ2zBvQ

**Review by wlanroot — APPROVED** [review]

File: constants/extraRpcs.js [hunk 1]
```diff
@@ -49,6 +49,8 @@ const privacyStatement = {
...
   jellypool:
     "The information collected by log files include internet protocol (IP) addresses, browser type, Internet Service Provider (ISP), date and time stamp, referring/exit pages, and possibly the number of clicks. These are not linked to any information that is personally identifiable. The purpose of the information is for analyzing trends, administering the site, tracking users' movement on the website, and gathering demographic information. https://www.jellypool.xyz/privacy/",
+  rpcfast:
+    "RPC Fast API is designed to prioritize user privacy, and therefore, we do not collect any personal information when clients interact with the blockchain through our provider. We do not track IP addresses, wallet addresses, executed transactions, or any other identifying data during interactions with our provider. For more info: https://rpcfast.com/privacy"
```
Comment (0xngmi) [inline constants/extraRpcs.js:53]: the privacy policy at https://rpcfast.com/privacy doesn't match whats on this line, instead it says
> We collect personally-identifying information that clients provide when subscribing to or operating on our Services.

**Comment by 0xngmi** [comment]
I applied the changes with the exception of the privacy stuff (left a comment on that) in commit 54bdbde

**Review by Karimsultan — APPROVED** [review]

**Comment by 0xngmi** [comment]
closing due to no new activity

**Review by Karimsultan — COMMENTED** [review]
T

**Pull request closed without merging**